actionpack/test/dispatch/session/cookie_store_test.rb

```diff
@@ -4,6 +4,7 @@
 require "stringio"
 require "active_support/key_generator"
 require "active_support/messages/rotation_configuration"
+require "fixtures/session_autoload_test/session_autoload_test/foo"
```

Comment:
I think "session_autoload_test" should not be required explicitly.

Reply:
At a glance I suspect it is caused by changing the behavior of `ActiveSupport::Dependencies.autoload_paths << path` in `with_autoload_path`:

https://github.com/rails/rails/blob/efaeca94e675504550973d0b79bc5c6bb07a2a83/actionpack/test/abstract_unit.rb#L160-L173

Reply:
Thanks for the review. Let me have a look at this.